### ISSUE TITLE
ibus-avro is not working ubuntu 14.04 

### ISSUE THREAD

**Issue by jisanB** [sarim/ibus-avro#108]
ibus-avro is not working ubuntu 14.04. i have all dependency

**Comment by saydulk** [comment]
yes, same here. release key also need to update http://linux.omicronlab.com/ubuntu_14.04.html

**Comment by jisanB** [comment]
same here

**Comment by sarim** [comment]
check this https://bn.maateen.me/install-avro-keyboard-in-ubuntu/
